libs/SalesforceSDK/src/com/salesforce/androidsdk/auth/HttpAccess.java

```diff
@@ -149,50 +138,21 @@ public synchronized OkHttpClient getOkHttpClient() {
     }
 
     /**
-     * Detects network changes and sets the network connectivity status.
-     *
-     * @param context The context of the request.
-     * @param intent Intent.
-     */
-    @Override
-    public void onReceive(Context context, Intent intent) {
```

Comment:
We don't need to monitor changes in network connectivity - we don't use them anywhere.